appletree/config.py

```diff
@@ -287,60 +290,82 @@ class SigmaMap(Config):
     def build(self, llh_name: Optional[str] = None):
         """Read maps."""
         self.llh_name = llh_name
-        if self.name in _cached_configs:
-            _configs = _cached_configs[self.name]
-        else:
-            _configs = self.get_default()
-            # Update values to sharing dictionary
-            _cached_configs[self.name] = _configs
-
-        if isinstance(_configs, dict):
-            try:
-                self._configs = _configs[llh_name]
-            except KeyError:
-                mesg = (
-                    f"You specified {self.name} as a dictionary. "
-                    f"The key of it should be the name of one "
-                    f"of the likelihood, but it is {llh_name}."
-                )
-                raise ValueError(mesg)
-        else:
-            self._configs = _configs
+        self._configs = self.get_configs()
```

Comment:
@xzh19980906 I think we do not need to set `_configs` as an attribute of `SigmaMap`.